auraed/src/cells/cell_service/cells/cell_info.rs

```diff
@@ -0,0 +1,12 @@
+use super::cgroups::cpu::CpuController;
+use super::cgroups::cpuset::CpusetController;
+use super::CellName;
+
+#[derive(Clone)]
+pub struct CellInfo {
```

Comment:
This looks like `CellSpec` + `CellName`. Do we need a dedicated struct for it?

Reply:
Good catch. Would it make sense to add the `CellName` to the `CellSpec` or should I replace the `CellInfo` struct with a tuple?

Reply:
I think it would be better as a tuple